### ISSUE TITLE
Enter, Escape keys not working at all

### ISSUE THREAD

**Issue by BadhanGanesh-HappyFox** [microsoft/vscode#117697]
Issue Type: <b>Bug</b>

I don't know, for some unknown reason, the Enter and Escape keys are not working at all.

This is getting really annoying as the basic of the basic things don't work at all. I can't rename symbols, can't go to next result when finding text in editor. Basically handicapped.

I am using VSCode to make iOS and Android apps using React Native. Even restarting the editor does not work sometimes. Pls fix this annoying bug. 

VS Code version: Code 1.53.2 (622cb03f7e070a9670c94bae1a45d78d7181fbd4, 2021-02-11T11:45:54.515Z)
OS version: Darwin x64 20.2.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i5-7360U CPU @ 2.30GHz (4 x 2300)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 6, 5|
|Memory (System)|8.00GB (0.10GB free)|
|Process Argv|--crash-reporter-id ef92d398-4fd7-48c9-b3e5-3ec67a3ef537|
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (41)</summary>

Extension|Author (truncated)|Version
---|---|---
rainbow-brackets|2gu|0.0.6
copy-relative-path|ale|0.0.2
code-gnu-global|aus|0.2.2
npm-intellisense|chr|1.3.1
path-intellisense|chr|2.3.0
bracket-pair-colorizer|Coe|1.0.61
bracket-pair-colorizer-2|Coe|0.2.0
vscode-svgviewer|css|2.0.0
vscode-markdownlint|Dav|0.39.0
vscode-eslint|dba|2.1.14
emulate|Die|1.4.0
es7-react-js-snippets|dsz|3.1.0
gitlens|eam|11.2.1
vscode-npm-script|eg2|0.3.16
prettier-vscode|esb|5.9.2
flow-for-vscode|flo|1.5.0
auto-close-tag|for|0.5.10
auto-complete-tag|for|0.1.0
auto-rename-tag|for|0.1.6
search-node-modules|jas|1.3.0
swift|Kas|0.2.0
rn-full-pack|kel|0.7.0
vscode-language-babel|mgm|0.0.30
vscode-clang|mit|0.2.3
cpptools|ms-|1.2.1
vscode-react-native|msj|1.4.0
color-highlight|nau|2.3.0
java|red|0.75.0
code-settings-sync|Sha|3.4.3
vscodeintellicode|Vis|1.2.11
vscode-java-debug|vsc|0.31.0
vscode-java-dependency|vsc|0.18.0
vscode-java-pack|vsc|0.12.1
vscode-java-test|vsc|0.28.0
vscode-maven|vsc|0.27.1
gitblame|wad|6.0.2
vscode-todo-highlight|way|1.0.4
change-case|wma|1.0.0
markdown-all-in-one|yzh|3.4.0
propertylist|zho|0.0.2
material-theme|zhu|3.9.15

(1 theme extensions excluded)

</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstry244:30256636
pythonvsdeb440:30248342
pythonvsded773:30248341
pythonvspyt875:30259475

```

</details>

**Comment by gjsjohnmurray** [comment]
Please use the Extension Bisect command to see if one of your extensions is causing this.

Also review the advice in https://github.com/microsoft/vscode/wiki/Keybinding-Issues

**Comment by OshotOkill** [comment]
I've got the same problem, vscode recognizes the `escape` key wrong and treat it as '🔼' (up key). I tried restart my bluetooth keyboard or reopen vscode but for a short period the editor even cannot detect `escape` key is pressed (no reaction).

I' m sure the `escape` key on my keyboard is working properly since the other editors are not having this issue, and I checked carefully about my keybindings and there's no conflict between `escape` and `🔼`. The issue just suddenly happens today.
